ivy-rich.el

```diff
@@ -209,7 +211,7 @@ Currently the transformed format is
             (format "%s%s%s"
                     filename
                     ivy-rich-switch-buffer-delimiter
-                    (ivy-rich-switch-buffer-pad path (- (window-width) (length filename)))))
+                    path))
```

Comment:
This line seems not related to issue, and is used for padding all the lines to same length. Can you describe the issue here?